opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -0,0 +1,349 @@
+#![allow(unused_imports, unused_mut, unused_variables, dead_code)]
```

Comment:
Could you create a tracking issue to resolve those warnings in a follow up PRs. I think we should clean those up before a new release

Reply:
Agree, created a tracking issue here : #1120

Reply:
Why not just clean them up before merging this? I don't see much value in postponing this work out of this PR.

Reply:
Sure, have fixed the warnings now. @TommyCpp -  fyi, if this is good to merge.